src/utils/MessageMixin.js

```diff
@@ -116,8 +116,13 @@ export default {
                     if (this.remainingTime !== 0 && this.active) {
                         this.remainingTime -= 1
                     } else {
-                        this.remainingTime = this.duration / 1000
-                        clearInterval(this.progress)
+                        /**
+                        * Wait until the component get closed and then reset
+                        **/
+                        window.setTimeout(() => {
+                            this.remainingTime = this.duration / 1000
+                            clearInterval(this.progress)
+                        }, 100)
```

Comment:
The component of notification get closed late after the progress-bar finish counting, so that occurs a visibility of the progress bar resetting, so I Have added the timeout function to wait 100ms until the component get closed completely.